epsagon/wrappers/fastapi.py

```diff
@@ -249,7 +326,25 @@ def wrapped_handler(*args, **kwargs):
             original_handler, request, status_code, args, kwargs
         )
 
-    dependant.call = wrapped_handler
+    async def async_wrapped_handler(*args, **kwargs):
+        """
+        Asynchronous wrapper handler
+        """
+        request: Request = _handle_wrapper_params(
+            args, kwargs, original_request_param_name
+        )
+        return await _async_fastapi_handler(
+            original_handler, request, status_code, args, kwargs
+        )
+
+    if is_async:
```

Comment:
better do:
if is async and IS_ASYNC MODE:
 ...
elif not is_async and not IS_ASYNC_MODE:
   ....